src/nginx/default.conf

```
@@ -1,12 +1,28 @@
 server {
-    listen 80;
-    server_tokens off;
-    server_name 127.0.0.1 ${APPLICATION_URL};
+    listen 80 default_server;
+    listen [::]:80 default_server;
+
+    location / {
+        return 301 https://$host$request_uri;
+    }
+}
+
+server {
+    listen 443 ssl http2 default_server;
+    listen [::]:443 ssl http2 default_server;
+
+    server_name ${APPLICATION_URL} ${DOMAIN_NAME};
 
     location /static/ {
         root /var/html/static/;
```

```suggestion
        root /var/html/;
```
Тут нужно убрать статик. У тебя работает потому, что я в докер контейнере создал папку и дубликат этого файла для теста и не удалил. При создании нового контейнера работать не будет. 
Впрочем, я удалил эту созданную папку и файл с сервера и скорее всего, при новом деплое уже не будет работать так.

---

Хорошо поправлю.